src/compiler/utilities.ts

```diff
@@ -4435,33 +4435,33 @@ namespace ts {
      * for example on a variable declaration whose initializer is a function expression.
      */
     export function hasJSDocParameterTags(node: FunctionLikeDeclaration | SignatureDeclaration): boolean {
-        return !!getFirstJSDocTag(node, SyntaxKind.JSDocParameterTag);
+        return !!getFirstJSDocTag(node, isJSDocParameterTag);
     }
 
     /** Gets the JSDoc augments tag for the node if present */
     export function getJSDocAugmentsTag(node: Node): JSDocAugmentsTag | undefined {

```

Comment:
@sandersn These functions are so simple now, and only used in a few places, maybe we should just inline them?

Reply:
Aren't they part of the public API? Otherwise I'd say yes.